### DISCUSSION TITLE
GBA Repro SAVING Issues

### DISCUSSION THREAD

**Discussion by judgeexodia** [sanni/cartreader#978] [Need help]
Hello,

I am having issues with being able to save correctly on GBA Repro Cartridges.  

I am using a Mother 3 Donor, Intel 4000L0YBQ0 (32MB) I believe.  I am successful in writing a game rom but when it comes to saving that is where I fail.  The board has no battery.  Originally the Save Type is: FLASH512_V131 (512Kbit).  

I am trying to write a homebrew pokemon game 16 MB.  It has the Save Type: FLASH1M_V103 (1024Kbit) .  

I'm unclear to how use the "Force Savetype".  I believe that may help.  (Not sure which option 512K SRAM/FRAM or 512K FLASH)

Getting error: 1M sub-circuit board is not installed. 
Tried GBA Tool Advance SRAM Patcher, but when I go to save it freezes the game.  

Do I need to expand the rom to fill the full 32 MB ? 

PLEASE HELP.  I've been stuck on this one all day.

**Comment by sanni** [comment]
"Force Savetype" just lets you manually choose the save type you want to dump ignoring the save type set by the database. It does not help you with your issue.

Closing this as this has nothing to do with the OSCR.

PS: You probably need a patcher like these: 
https://github.com/bbsan2k/Flash1M_Repro_SRAM_Patcher   
https://github.com/metroid-maniac/gba-auto-batteryless-patcher/
